client/app/cda/details/cda-details.controller.js

```diff
@@ -29,10 +27,22 @@ angular.module("managerApp")
             self.messageHandler = CloudMessage.subscribe("paas.cda", { onMessage: () => self.refreshMessage() });
         };
 
+        self.loadGuides = function () {
+            return OvhApiMe.v6()
+                .get().$promise
+                .then(me => {
+                    self.guides.list[0].url = URLS.guides.cda[me.ovhSubsidiary];
+                })
```

Comment:
ovhDocUrl has been precisely made to prevent the usage of URLS.
Don't we have another choice?

Reply:
It's a 2API service and they are supposed to be replaced soonish so I don't know, I don't think we should bet on such things 😃